CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_ad_reporting v1.2.1
```

Comment:
Since we are updating the LinkedIn Ads package to use the latest breaking change, should this also be a breaking change?